sdl2-vga-terminal/src/VgaTerminal.hpp

```diff
@@ -13,16 +13,13 @@ class VgaTerminal : public Window
     typedef struct
     {
         uint8_t  mode; // video mode (only mode 3 available at the moment)
-        //uint16_t sw;   // screen   width
-        //uint16_t sh;   //          height
         uint8_t  tw;   // terminal width
         uint8_t  th;   //          hieght
         uint8_t  cw;   // char     width
         uint8_t  ch;   //          height  | font size
-        //uint8_t  fs;   // font size 
         uint8_t* font;
         int  numColors;
-        uint8_t* palette; // BGR palette assumed (might be required a palette format flag?)
+        uint8_t* palette; // RGB palette assumed (might be required a palette format flag?)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct member 'videoMode_t::palette' is never used.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5379806)

Reply:
?